services/kingdom_getters.js

```diff
@@ -0,0 +1,45 @@
+const fs = require("fs");
+
+const getJson = () => {
```

Comment:
This function is copy-pasted from the other file. I'd recommend extracting it to some kind of `utils` module.